src/backends/kimchi/mod.rs

```diff
@@ -124,7 +126,10 @@ impl Witness {
 
     pub fn debug(&self) {
         for (row, values) in self.0.iter().enumerate() {
-            let values = values.iter().map(|v| v.pretty()).join(" | ");
+            let values = values
+                .iter()
+                .map(super::super::helpers::PrettyField::pretty)
```

Comment:
`super::super` o_O? just use `crate::helpers::`

Reply:
Ah, this was an automatic clippy fix. Changed to `PrettyField::pretty`